src/types/paragraph.ts

```diff
@@ -70,8 +75,8 @@ export interface ParagraphAlertSingle extends DrupalParagraph {
 
 export interface ParagraphAudienceTopics extends DrupalParagraph {
   field_audience_selection: string | null
-  field_audience_beneficiares: TaxonomyTermAudienceBeneficiares
-  field_non_beneficiares: TaxonomyTermAudienceNonBeneficiares
+  field_audience_beneficiares: TaxonomyTermAudienceBeneficiaries
```

Comment:
This is correct. The Taxonomy/Vocabulary is named correctly. `field_audience_beneficares` is correct in terms of what the machine name is, even though it is spelled incorrectly.